legacy/herbie-old/wsgi.py

```diff
@@ -1,5 +1,5 @@
 """
-WSGI config for herbie project.
```

Comment:
is the name correct here?` herbie-old` or these files must be removed later?

Reply:
Yes I will remove them once the PR is ready.